benchexec/tools/legion-symcc.py

```diff
@@ -41,5 +44,12 @@ def cmdline(self, executable, options, task, rlimits):
     def get_value_from_output(self, output, identifier):
         for line in reversed(output):
             if line.startswith(identifier):
-                return line[len(identifier) :]
+                value = line[len(identifier) :]
+                return value.strip()
         return None
+
+    def determine_result(self, run):
+        for line in run.output:
+            if "reach_error() detected." in line:
+                return result.RESULT_FALSE_REACH
+        return result.RESULT_UNKNOWN
```

Comment:
Is "unknown" desired for all other results? Previously it was "done".

The difference is supposed to be as follows: If the tool is given a task that can be answered with true/false (e.g., "does this property hold?"), then "unknown" is the answer for "don't know". If the tool is just given the task to do something (no actual answer expected), then "done" is the answer for "completed successfully".